types/type-specs.tsx

```diff
@@ -51,8 +52,14 @@ const invalidParams: Params = { id: 13 }; // $ExpectError
 /*
  * Link and Redirect component type specs
  */
+
+// `to` and `href` are aliases, but they are mutually exclusive and
+// can't be used at the same time:
 <Link to="/users">Users</Link>;
 <Link href="/about">About</Link>;
+<Link href="/about" to="/app" children="" />; // $ExpectError
```

Comment:
delete

Reply:
Why should we delete those? These lines assert the error: when both `href` and `to` are given TypeScript will throw an error. The `// $ExpectError` is a magic comment that tells `dtslint` that this lines **should raise** a type error.